boards/xtensa/esp32s3/common/src/esp32s3_lan9250.c

```diff
@@ -62,6 +62,12 @@ static void lan9250_getmac(const struct lan9250_lower_s *lower,
  * Private Data
  ****************************************************************************/
 
+#ifdef CONFIG_LAN9250_SPI
+  static struct spi_dev_s *g_dev;
+#else
+  static struct qspi_dev_s *g_dev;
```

Comment:
Why all the other LAN that not LAN9250 are supposed to be QSPI? It doesn't make much sense

Reply:
Ok, I miunderstood that! Actually it was only for LAN9250 itself that could work on SPI or QSPI modes